crates/bevy_pbr/src/meshlet/meshlet_bindings.wgsl

```diff
@@ -7,23 +7,19 @@
 struct PackedMeshletVertex {
     a: vec4<f32>,
     b: vec4<f32>,
-    tangent: vec4<f32>,
 }
 
-// TODO: Octahedral encode normal, remove tangent and derive from UV derivatives
 struct MeshletVertex {
     position: vec3<f32>,
     normal: vec3<f32>,
     uv: vec2<f32>,
```

Comment:
Could use f16 for the uvs

Reply:
I've heard very sus things about precision issues with f16 UVs, especially given the large/detailed meshes I intend to use this with. So I'm very hesitant to adopt f16 UVs, but we'll see.